README.rst

```diff
@@ -13,9 +13,6 @@
 .. image:: https://raster.shields.io/badge/Follows-NEP29-brightgreen.png
     :target: https://numpy.org/neps/nep-0029-deprecation_policy.html
 
-.. image:: https://tidelift.com/subscription/pkg/pypi-ipython
-    :target: https://tidelift.com/badges/package/pypi/ipython?style=flat
```

Comment:
I think it's not broken, just the 2 urls are swapped.

Reply:
Thanks @Carreau 
Created a new PR #13793 now its working fine.

Reply:
Thanks. Generally you do not need to create a new PR, you can just push on the same branch and edit the existing one. Closing Here, I'll merge the other.